cmd/genji/dbutil/dump.go

```diff
@@ -64,10 +64,86 @@ func Dump(ctx context.Context, db *genji.DB, w io.Writer, tables ...string) erro
 	return err
 }
 
-// dumpTable displays the content of the given table as SQL statements.
-func dumpTable(tx *genji.Tx, tableName string, w io.Writer) error {
+// dumpTableContent displays the content of the given table as SQL statements.
+func dumpTableContent(tx *genji.Tx, w io.Writer, tableName string) error {
+	// Dump schema first.
+	if err := dumpSchema(tx, w, tableName); err != nil {
+		return err
+	}
+
+	t, err := tx.GetTable(tableName)
+	if err != nil {
+		return err
+	}
+
+	q := fmt.Sprintf("SELECT * FROM %s", t.Name())
+	res, err := tx.Query(q)
+	if err != nil {
+		return err
+	}
+	defer res.Close()
+
 	var buf bytes.Buffer
 
+	// Inserts statements.
+	insert := fmt.Sprintf("INSERT INTO %s VALUES ", t.Name())
+	return res.Iterate(func(d document.Document) error {
+		buf.WriteString(insert)
+
+		data, err := document.MarshalJSON(d)
+		if err != nil {
+			return err
+		}
+
+		buf.Write(data)
+		buf.WriteString(";\n")
+		_, err = buf.WriteTo(w)
```

Comment:
Using a bytes.Buffer to then write to w is useless here, instead we can replace it by a bufio.Writer https://golang.org/pkg/bufio/#Writer or fmt.Fprintf (see DumpSchema for an example)

Reply:
Yes good catch !